go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/pingcap/fn v0.0.0-20191016082858-07623b84a47d
 	github.com/pingcap/goleveldb v0.0.0-20191226122134-f82aafb29989
 	github.com/pingcap/kvproto v0.0.0-20200828054126-d677e6fd224a
-	github.com/pingcap/log v0.0.0-20200828042413-fce0951f1463
+	github.com/pingcap/log v0.0.0-20200511115504-543df19646ad
```

Comment:
Unexpected change?

Reply:
I'm not sure. Since I noticed that it will change everytime I run `make test`.

Should I just ignore it?

Reply:
If you want to test on your laptop, you can use `make gotest`.